esp-hal/src/spi/master.rs

```diff
@@ -841,56 +839,33 @@ pub mod dma {
         }
     }
     /// An in-progress DMA transfer
-    pub struct SpiDmaTransferRxTx<'d, T, C, RBUFFER, TBUFFER, M>
+    pub struct SpiDmaTransferRxTx<'t, 'd, T, C, M>
```

Comment:
We should mark all the `Transfer` structs with `#[must_use]`.

Reply:
Good idea! 👍